### ISSUE TITLE
Massive problems with way to low speed 

### ISSUE THREAD

**Issue by MRNOBODY515** [yumi-modding/FS22_CombineXP#43]
Hey Yumi!
I have big issues with the speed in combine XP. I don't know why. But it seems Giants has changed something. For example. No Mods. Erlengrad. Fendt 9T with the 30ft (9,2m) cutter. Over 250% in realistic at 4km/h.  For around 100% I need to slow down to 1 km/h.
In combination with Precission Farming unplayable. 1km/h is way to much. Can you take a look at this?

Now I have reloaded the test savegame and it works ans I can drive normal. I really don't understand this...

**Comment by yumi-modding** [comment]
Hello @MRNOBODY515 
Can you please check at what time you're harvesting? If it's in the early morning and the option "Daytime limit" is activated in the settings, this is a "normal" behavior that is supposed to mimic moisture to slow down harvesting in the early morning.
You can either harvest later in the day or disable this setting.

**Comment by MRNOBODY515** [comment]
I've started at 10am on my Savegame. All the time 300-400% with 1km/h.The combine stops with activated cruise control. Then I wait till 12 o clock. The same. Disabled daytime limit. Changed from realistic to arcade.. Another power configuration with more power and so on. Disable Precission Farming. Nothing helps. No log issues. 
Then all mods removed. Started new Savegame as written. Same issue. Reloaded the new Savegame. Issue is gone. Back to my normal Savegame. Issue is still gone. Absolutely curious and at the moment and I dont know how to reproduce.
If I could find out how this happens I will tell you. For the moment it can be closed. Thanks for your help!
